CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.17)
 
-project(impylib LANGUAGES Fortran)
+project(chromolib LANGUAGES Fortran)
```

Comment:
This should also be called chromo, I think.

Reply:
Sure, why not.